data/card_catalogue/upgradeable-1.json

```diff
@@ -408,49 +408,36 @@
     "tier": "blue",
     "details": {
       "rules": {
-        "active": "You can attack two additional times on your turn.",
+        "active": "Make two instant attacks.",
         "passive": null
       },
-      "flavour": "Before they even realized what happened, another punch sent them flying.",
+      "flavour": "The first blow shatters their defenses, the next their spirit.",
       "image": "res/art/illustration/Double Strike.png"
     }
   },
   {
     "title": "Quad Strike",
-    "upgrade": "Strike Storm",
+    "upgrade": "Endless Blows",
     "tier": "green",
     "details": {
       "rules": {
-        "active": "You can attack four additional times on your turn.",
+        "active": "Make four instant attacks",
         "passive": null
       },
       "flavour": "Their defense won't last forever. Strike, and strike again.",
       "image": "res/art/illustration/Quad Strike.png"
     }
   },
-  {
-    "title": "Strike Storm",
-    "upgrade": "Endless blows",
-    "tier": "red",
-    "details": {
-      "rules": {
-        "active": "You can make twice times as many attacks as you otherwise would on your turn.",
```

Comment:
Doubling is always exciting, but I think that the take X attack X times will lead to crazier attacks in more situations